aliases_test.go

```diff
@@ -137,3 +137,27 @@ func TestDeleteAliasNotFound(t *testing.T) {
 		t.Error("expected number of errors to be > 1")
 	}
 }
+
+func TestGetAliasableResource(t *testing.T) {
+	requests := []autopilot.TestRequest{
+		autopilot.NewTestRequest(
+			``,
+			`{}`,
+			`{}`,
+		),
+		autopilot.NewTestRequest(
+			``,
+			`{}`,
+			`{}`,
+		),
+	}
+	client := BestTestClient(t, "tags/get_aliasable_resource", requests...)
+	// Act
+	service, err1 := client.GetAliasableResource(ol.AliasOwnerTypeEnumService, "")
```

Comment:
With this setup i should beable to test each branch of the switch statement in 1 go